go.mod

```diff
@@ -75,6 +75,8 @@ require (
 	github.com/mattn/go-sqlite3 v1.11.0
 	github.com/mcuadros/go-version v0.0.0-20190308113854-92cdf37c5b75
 	github.com/microcosm-cc/bluemonday v0.0.0-20161012083705-f77f16ffc87a
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
```

Comment:
Please use go1.13 to compile.

Reply:
I upgraded, but when I did a `make generate` and `make build` the go.sum file remained the same. So I just reverted back to master's go.sum and built again and works. Committed the reverted go.sum file. Was that the right thing to do @lunny?

Reply:
I think that happens when you update the tools as well (e.g. golanci-lint). Try building again and see if it changes again.